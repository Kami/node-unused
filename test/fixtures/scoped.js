// the foo at global scope remains unused
// when a new foo inside bar is declared

var foo = 'bz';

function bar() {
    var foo = 'dog';
    foo = 'cat';
}

bar();
