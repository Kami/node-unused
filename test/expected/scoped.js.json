[
  {
    "name": "foo",
    "loc": {
      "line": 4,
      "column": 4
    },
    "param": false
  }
]